pkg/config/config.go

```diff
@@ -32,6 +33,22 @@ type ComponentSettings struct {
 	// The builder image to use
 	ComponentType *string `yaml:"ComponentType,omitempty"`
```

Comment:
Could we use just Type here?

Reply:
I prefer the verbosity

Reply:
In this case, it is not verbosity but duplicity ;-) the whole type is called `ComponentSettings`

Reply:
Addressed as part of #1453